src/SFA.DAS.AdminService.Web.UnitTests/Models/WhenBuildingError403ViewModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Models;
+
+namespace SFA.DAS.AdminService.Web.UnitTests.Models
+{
+    public class WhenBuildingError403ViewModel
+    {
+        [Test]
+        public void Then_The_HelpPage_Link_Is_Correct_For_Production_Environment()
+        {
+            var actual = new Error403ViewModel("prd");
+
+            Assert.That(actual.HelpPageLink, Is.Not.Null);
+            Assert.That(actual.HelpPageLink, Is.EqualTo("https://services.signin.education.gov.uk/approvals/select-organisation?action=request-service"));
+        }
+
+        [Test]
+        public void Then_The_HelpPage_Link_Is_Correct_For_Non_Production_Environment()
+        {
+            var actual = new Error403ViewModel("test");
+
+            Assert.That(actual.HelpPageLink, Is.Not.Null);
+            Assert.That(actual.HelpPageLink, Is.EqualTo("https://test-services.signin.education.gov.uk/approvals/select-organisation?action=request-service"));
+        }
+
+        [TestCase("")]
+        [TestCase(null)]
+        public void Then_The_HelpPage_Link_Is_Correct_When_Environment_Is_Null(string env)
+        {
+            var actual = new Error403ViewModel(env);
+
+            Assert.That(actual.HelpPageLink, Is.Not.Null);
+            Assert.That(actual.HelpPageLink, Is.EqualTo("https://services.signin.education.gov.uk/approvals/select-organisation?action=request-service"));
```

Comment:
Same thing for these hardcoded URLs. The tests don't need to worry about the actual URL, only that whatever URL retrieved from the config is being used. You can do this by mocking the `HelpPageLink` property in your view model

Reply:
Changed the help link from configuration. So this change is not required anymore.